samples/server/petstore/aspnetcore/src/Org.OpenAPITools/Controllers/UserApi.cs

```diff
@@ -32,8 +33,7 @@ public  class UserApiController : ControllerBase
         /// <param name="body">Created user object</param>
         /// <response code="0">successful operation</response>
         [HttpPost]
-        [Route("/v2/user")]
-        [ValidateModelState]
+        [Route("/v2/user")]        [ValidateModelState]
```

Comment:
@MBcom Looks like the line break has been incorrectly removed?

Reply:
oh correct - i fixed it - i didn't notice it yet because it's not a syntax error :)